runtime/viewlet.js

```diff
@@ -79,6 +80,11 @@ class Viewlet {
   }
 }
 
+/** @class View
+ * A handle on a set of Entity data. A particle's manifest dictates
```

Comment:
Is `set` enough to imply no duplicates/ordering?

Reply:
I think so?